stairway/src/test/java/bio/terra/stairway/impl/EnumerateFlightsTest.java

```diff
@@ -79,10 +81,15 @@ public void enumTest() throws Exception {
     flightList = flightDao.getFlights(0, 100, filter);
     checkResults("case 2", flightList, Arrays.asList("0", "1", "2", "3", "4", "5"));
 
-    // Case 3: date with null values - form1
+    // Case 3.1: date with values - form1
     filter = new FlightFilter().addFilterCompletedTime(FlightFilterOp.GREATER_THAN, minSubmit);
     flightList = flightDao.getFlights(0, 100, filter);
-    checkResults("case 3", flightList, Arrays.asList("0", "1", "2"));
+    checkResults("case 3.1", flightList, Arrays.asList("0", "1", "2"));
```

Comment:
FWIW we've been migrating our code from `Arrays.asList()` to `List.of()`, which is the "new" way to create lists.
```suggestion
    checkResults("case 3.1", flightList, List.of("0", "1", "2"));
```